### PR TITLE
[Abyssea] Fix behavior for coffer and split files for sturdypyxis

### PR DESCRIPTION
**Pull request by Chronos77**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: scripts/commands/addlights.lua [hunk 1]
```diff
@@ -62,5 +62,5 @@ function onTrigger(player,light,amount,target)
 
     xi.abyssea.addPlayerLights(targ, setLight, amount)
     local newAmount = targ:getCharVar(light.."Light")
-    player:PrintToPlayer(string.format("%s was given %i %s light, for a total of %i.",targ:getName(),amount,light,newAmount))
+    player:PrintToPlayer(string.format("%s was given %i %s light, for a total of %i.", targ:getName(), amount, light, newAmount))
 end
```
Comment (Xaver-DaRed) [inline scripts/commands/addlights.lua]: Could you add a newline here?

File: scripts/globals/abyssea/sturdypyxis/augmented_item.lua [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+    local item1aug1val = GetAugVal(npc, 1, 1)
+    local item1aug2    = GetAug(npc, 1, 2)
+    local item1aug2val = GetAugVal(npc, 1, 2)
+
+    local item2aug1    = GetAug(npc, 2, 1)
+    local item2aug1val = GetAugVal(npc, 2, 1)
+    local item2aug2    = GetAug(npc, 2, 2)
+    local item2aug2val = GetAugVal(npc, 2, 2)
+
+    if (slot == 1) then
```
Comment (Xaver-DaRed) [inline scripts/globals/abyssea/sturdypyxis/augmented_item.lua]: These parenthesis ( ) are not necesary. A couple more right below in this file, but i don't want to spam you.

File: scripts/globals/abyssea/sturdypyxis/blue_chest.lua [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+            npc:setLocalVar("FAILED_ATTEMPTS", failedAttempts)
+
+            if failedAttempts >= attemptsallowed then
+                xi.pyxis.removeChest(player, npc, 0, 1)
+                xi.pyxis.messageChest(player, ID.text.PLAYER_FAILED_LOCK, 0, 0, 0, 0, npc)
+                player:messageSpecial(ID.text.CHEST_DISAPPEARED)
+            end
+        end
+    end
+end
```
Comment (Xaver-DaRed) [inline scripts/globals/abyssea/sturdypyxis/blue_chest.lua]: New line at the end, please.

File: scripts/globals/abyssea/sturdypyxis/cruor.lua [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    local ID = zones[npc:getZoneID()]
+    local alliance = player:getAlliance()
+    local cruorAmount = npc:getLocalVar("CRUOR")
+    for p, member in ipairs(alliance) do
+        if member:getZoneID() == player:getZoneID() and member:isPC() then
+            member:addCurrency("cruor", cruorAmount)
+            member:messageSpecial(ID.text.CRUOR_OBTAINED, cruorAmount)
+        end
+    end
+end
```
Comment (Xaver-DaRed) [inline scripts/globals/abyssea/sturdypyxis/cruor.lua]: And new line here aswell. Same with next 3 files in the list.

File: scripts/globals/abyssea/sturdypyxis/npc.lua [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+                end,
+                [xi.pyxis.chestDropType.KEY_ITEM] = function() -- KI
+                    xi.pyxis.ki.giveKeyItem(player, npc)
+                end,
+            }
+        elseif openchoice == 2 then
+            xi.pyxis.removeChest(player, npc, 1, 1)
+        end
+    end
+end
```
Comment (Xaver-DaRed) [inline scripts/globals/abyssea/sturdypyxis/npc.lua]: newline

File: scripts/globals/abyssea/sturdypyxis/popitem.lua [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+        GiveItem(player, npc, itemSelected)
+    elseif itemSelected == 9 then
+        for _, v in ipairs(loottable) do
+            player:addTreasure(v)
+        end
+
+        xi.pyxis.messageChest(player, ID.text.ADD_SPOILS_TO_TREASURE, 0, 0, 0, 0, npc)
+        xi.pyxis.removeChest(player, npc, 0, 1)
+    end
+end
```
Comment (Xaver-DaRed) [inline scripts/globals/abyssea/sturdypyxis/popitem.lua]: newline in this and next 2 files

File: scripts/globals/abyssea/sturdypyxis/time.lua [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+            member:messageSpecial(ID.text.ABYSSEA_TIME_OFFSET + 3, 10, 1)
+            local effect = member:getStatusEffect(xi.effect.VISITANT)
+            local oldDuration = tonumber(effect:getTimeRemaining())
+            local newDuration = (oldDuration + tonumber(10 * 60)) * 1000
+            effect:setDuration(newDuration)
+            effect:resetStartTime()
+            effect:setIcon(xi.effect.VISITANT)
+        end
+    end
+end
```
Comment (Xaver-DaRed) [inline scripts/globals/abyssea/sturdypyxis/time.lua]: newline

**Comment by Xaver-DaRed** [comment]
This looks solid. But Clay has to be the one to review it.

File: scripts/globals/abyssea/sturdypyxis/time.lua (hunk 7 above)
Comment (Chronos77) [reply scripts/globals/abyssea/sturdypyxis/time.lua]: Done !

File: scripts/globals/abyssea/sturdypyxis/popitem.lua (hunk 6 above)
Comment (Chronos77) [reply scripts/globals/abyssea/sturdypyxis/popitem.lua]: Done !

File: scripts/globals/abyssea/sturdypyxis/npc.lua (hunk 5 above)
Comment (Chronos77) [reply scripts/globals/abyssea/sturdypyxis/npc.lua]: Done!

File: scripts/globals/abyssea/sturdypyxis/cruor.lua (hunk 4 above)
Comment (Chronos77) [reply scripts/globals/abyssea/sturdypyxis/cruor.lua]: Done !

File: scripts/globals/abyssea/sturdypyxis/blue_chest.lua (hunk 3 above)
Comment (Chronos77) [reply scripts/globals/abyssea/sturdypyxis/blue_chest.lua]: done !

File: scripts/globals/abyssea/sturdypyxis/augmented_item.lua (hunk 2 above)
Comment (Chronos77) [reply scripts/globals/abyssea/sturdypyxis/augmented_item.lua]: done !

File: scripts/commands/addlights.lua (hunk 1 above)
Comment (Chronos77) [reply scripts/commands/addlights.lua]: done !

**Review by claywar — APPROVED** [review]

**Pull request merged**